formats/json-jackson/src/main/java/io/cloudevents/jackson/CloudEventDeserializer.java

```diff
@@ -176,7 +177,7 @@ private String getStringNode(ObjectNode objNode, JsonParser p, String attributeN
 
         private String getOptionalStringNode(ObjectNode objNode, JsonParser p, String attributeName) throws JsonProcessingException {
             JsonNode unparsedSpecVersion = objNode.remove(attributeName);
-            if (unparsedSpecVersion == null) {
+            if (unparsedSpecVersion == null || unparsedSpecVersion instanceof NullNode) {
```

Comment:
not your fault, but shouldn't the variable `unparsedSpecVersion` have a more generic name? Otherwise it seems like something is done with the specVersion

Reply:
+1

Reply:
Done https://github.com/cloudevents/sdk-java/pull/432/commits/f543e00368b30388771392b73a4046d39143fc31